packages/components/src/date-time/index.js

```diff
@@ -31,6 +32,7 @@ function DateTimePicker(
 	},
 	ref
 ) {
+	const { __, _x } = useI18n();
```

Comment:
@jsnajdr – is it something that the WP-CLI command is able to understand when extracting translations?

Reply:
Yes, we are already using this convention in some WordPress.com projects like Gutenboarding and Editing Toolkit Plugin and `wp-cli` is able to extract the translation.

However, there's been a lot of pain trying to extract translations from published minified plugin JS files. The `__` variable gets mangled sometimes and one needs to configure Terser to not mangle these:
```js
terserOptions: {
  mangle: { reserved: [ '__', '_n', '_nx', '_x' ] },
}
```
But even then it's not reliable -- webpack module concatenation can rename variables, too. If there are two modules that do something like:
```js
import { __ } from '@wordpress/i18n';

__( 'Hello' );
```
and they are concatenated into one module, one of the `__` variables will be renamed.

But the minification issues are not unique to the `react-i18n` library -- they happen with the classic `i18n`, too. Just wanted to point out something we've been fighting with when extracting translations with `wp-cli`.

Reply:
Webpack handling was improved in Gutenberg some time ago so it should be all good:
https://github.com/WordPress/gutenberg/blob/068487d568c05367837e9364df8414268fe574c2/webpack.config.js#L84-L86